constructor/construct.py

```diff
@@ -15,26 +15,42 @@
 
 # list of tuples (key name, required, type, description)
 KEYS = [
-    ('name',                   True,  str, '''
+    (
```

Comment:
I was thinking of turning `KEYS` into a proper `pydantic` schema instead of this homemade pseudo-validation, in a future release. I am inclined to ignore changes in this file for now.


They are only formatting changes so that's fine but my thinking is that this file won't last as is for too long.